### ISSUE TITLE
console.log jumping out on promise functions

### ISSUE THREAD

**Issue by jrayga** [Chakroun-Anas/turbo-console-log#114]
```
  function getUsers() {
        $scope.isLoadingData = true
        var promise = UsersFactory.getUsers()
        promise.then(function (data) {
            $scope.users= data.data.result;
            $scope.isLoadingData = false
        }).catch(function (error) {
            $scope.isLoadingData = false
        })
        console.log("🚀 ~ file: Users.js ~ line 151 ~ $scope.users", $scope.users)
   }

    console.log("🚀 ~ file: Users.js ~ line 154 ~ error", error)
```

Coded in AngularJS 1.8.0.

So when I try to use TCL on this function the console.log jumps out on the promise.then statement,
Also when I tried to use TCL on the error variable in the promise.catch statement the console.log jumps outside the getUsers function,

Is this a problem with AngularJS or maybe some of my extensions intervene with TCL?

Extensions list

- bmewburn.vscode-intelephense-client
- ChakrounAnas.turbo-console-log
- Dart-Code.dart-code
- Dart-Code.flutter
- dracula-theme.theme-dracula
- dsznajder.es7-react-js-snippets
- eiminsasete.apacheconf-snippets
- felixfbecker.php-intellisense
- Gruntfuggly.todo-tree
- hangxingliu.vscode-nginx-conf-hint
- johnpapa.Angular2
- MikeBovenlander.formate
- ms-edgedevtools.vscode-edge-devtools
- ms-python.python
- ms-toolsai.jupyter
- ms-vscode-remote.remote-wsl
- Nash.awesome-flutter-snippets
- PKief.material-icon-theme
- PrimaFuture.open-php-html-js-in-browser
- raynigon.nginx-formatter
- streetsidesoftware.code-spell-checker
- thekalinga.bootstrap4-vscode
- toba.vsfire
- william-voyek.vscode-nginx
- yzhang.markdown-all-in-one
- ZainChen.json

**Comment by glaucopater** [comment]
Same issue for me

**Comment by ImanMahmoudinasab** [comment]
I have it as well.

**Comment by iamkabirmalik** [comment]
same for me

**Comment by Chakroun-Anas** [comment]
Should be fixed in version `2.6.2` and  above, can you check again please @jrayga @glaucopater @iamkabirmalik @ImanMahmoudinasab  and give me your feedback ? Thx
